README.md

```diff
@@ -1,38 +1,69 @@
-# This is a MkDocs project
+# Welcome to Allfeat Documentation! 🚀
 
-## Install
+Welcome to the Allfeat documentation! This guide is your gateway to understanding and leveraging Allfeat's cutting-edge platform for decentralized applications (DApps) and smart contracts. Whether you're a developer, validator, or simply curious about the possibilities of decentralized music industry infrastructure, you've come to the right place! 💿🎶
 
-### Prerequisites
+## What is Allfeat? 🎵
```

Comment:
This part isn't required to appear in a README I guess, Allfeat explanation is already in the doc